Pod/Classes/DatabaseConnection.swift

```diff
@@ -12,6 +12,8 @@ import sqlite3
 /** Valid SQLite types are marked using the 'SQLiteValue' protocol */
 public protocol SQLiteValue {}
 
+extension NSArray: SQLiteValue {}
+extension NSDictionary: SQLiteValue {}
```

Comment:
i have try to defined 

`extension NSArray: SQLiteValue {}`

`extension NSDictionary: SQLiteValue {}`

in PropertyData.swift. but error

`PropertyData.swift:10:20: Redundant conformance of 'NSArray' to protocol 'SQLiteValue'`

`PropertyData.swift:11:25: Redundant conformance of 'NSDictionary' to protocol 'SQLiteValue'`

You can try it